src/components/Input.tsx

```diff
@@ -1,17 +1,27 @@
 import { Input } from "antd";
 import styles from "@/styles/Chat.module.css";
-import { AudioFilled, SendOutlined, LoadingOutlined, CloseOutlined } from "@ant-design/icons";
+import {
+  AudioFilled,
+  SendOutlined,
+  LoadingOutlined,
+  CloseOutlined,
+} from "@ant-design/icons";
 import { useEffect, useState } from "react";
+import { createSpeechlySpeechRecognition } from "@speechly/speech-recognition-polyfill";
 import SpeechRecognition, {
   useSpeechRecognition,
 } from "react-speech-recognition";
+import { APP_ID } from "@/utils/constants";
 
 interface InputProps {
   sendMsg: (msg: string) => void;
   lang: string;
   finishLoading: boolean;
   finishTyping: boolean;
 }
+const appId = APP_ID;
+const SpeechlySpeechRecognition = createSpeechlySpeechRecognition(appId);
```

Comment:
can we add this code snippet only if the browser doesn't support the library line 42